CONTRIBUTING.md

```diff
@@ -9,10 +9,16 @@ We are actively encouraging users to ask questions and start discussions in the
 [discussions tab] of this repository. Does something seem like it's broken?
 Please go ahead and [open an issue]!
 
-The [website] pages are open to contributions but they will need to be reviewed
-by a member or associate member of ARC. We might be slow to approve new tool
-suggestions (since we'll probably want to discuss them first) but don't let that
-put you off creating an issue.
+The template and [website] pages are open to contributions! But they will need
+to be reviewed by a member or associate member of ARC.
+
+We will probably be slow to approve new tool suggestions (since we'll probably
+want to discuss them first) but don't let that put you off creating an issue!
+
+Any opinion changes should be approved by at least two reviewers who are
```

Comment:
```suggestion
Any opinionated changes should be approved by at least two reviewers who are
```

For consistency with usage below and as `opinion changes` reads a bit oddly to me, but I'm not strong opinionated on this 🙃

Reply:
I guess maybe the key point here is do we mean changes in the opinion reflected in the website / template (in which case I think current text is fine) or any changes to repository we believe are 'opinionated' which I'm interpreting as due to subjective preferences rather than something more 'objective' (admittedly that distinction is itself quite subjective...).